src/components/TransactionPreview/HistoryPreview.tsx

```diff
@@ -70,7 +70,7 @@ const HistoryPreview = ({ crossChainAction }: TransactionPreviewInterface) => {
     let gasFees = gasInETHAmount * crossChainAction.estimated.usdPrice;
     let gasFeesNative = gasInETHAmount;
 
-    gasFeesNative = gasInETHAmount.toFixed(4) < '0.0001' ? '>0.0001' : gasInETHAmount.toFixed(5);
+    gasFeesNative = gasInETHAmount.toFixed(4) < '0.0001' ? '<0.0001' : gasInETHAmount.toFixed(5);
```

Comment:
make this comparison as numerical, comparing strings can lead to unwanted results